Src/Campus.Services.Interfaces/DTO/User/UserViewDto.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Campus.Services.Interfaces.DTO.User
+{
+    public class UserViewDto
+    {
+        public string Email { get; set; }
+        public string UserName { get; set; }
```

Comment:
Is UserName field intended for holding Nickname ?, If so, consider changing to Nickname

Reply:
The `UserName` is default field in the `Core Identity`